remodel/decorators.py

```diff
@@ -1,4 +1,4 @@
-class cached_property(object):
+class CachedProperty(object):
```

Comment:
Since they are decorators, I prefer them in underscore_case, just like the rest of decorators out there (same applies to similar changes below). Also, some of these will find their way out of the code base, once I finish the refactoring.